### DISCUSSION TITLE
Row Group Aggregation

### DISCUSSION THREAD

**Discussion by vitolpoc** [infinite-table/infinite-react#21] [Q&A] [answered]
Is it possible to configure a table with groups and no pivot columns so that the columns display the aggregation for the row groups ?

In the following sandbox, I configured the table with row groups (language & stack), the salary column and a salary aggregation.
https://codesandbox.io/s/infinite-table-group-aggregation-94o7d?file=/src/App.tsx

Actual Result:
- every row groups are expandable
- salary values are visible when expanding the stack group nodes

Expected Result:
- every row should display the average salary by row group (language and stack)
- the stack group nodes should not be expandable (or we may want to configure if it's expandable or not)

**Comment by roblotter** [comment, marked as answer]
See https://infinite-table.com/docs/latest/learn/grouping-and-pivoting/grouping-rows#aggregations for aggregations working together with row grouping

For overriding the expandable state of certain row groups, please see a demo [here](https://codesandbox.io/s/react-infinite-table-row-grouping-custom-last-group-render-dmpuq)
